au4/src/trackedit/internal/trackeditactionscontroller.cpp

```diff
@@ -338,7 +339,10 @@ void TrackeditActionsController::paste()
     TrackId selectedTrackId = selectionController()->selectedTrack();
 
     if (!tracks.empty() && selectedStartTime >= 0) {
-        trackeditInteraction()->pasteFromClipboard(selectedStartTime, selectedTrackId);
+        auto ret = trackeditInteraction()->pasteFromClipboard(selectedStartTime, selectedTrackId);
+        if (ret.code() == static_cast<int>(trackedit::Err::StereoClipIntoMonoTrack)) {
```

Comment:
Are we deliberately not showing other types of errors?

Reply:
I did it on purpose at first but I checked AU3 behaviour now and I guess we should show them all, will fix